Packages/Testing-MIES/UTF_Epochs.ipf

```diff
@@ -108,6 +108,7 @@ static Function TestEpochChannelTight(e)
 End
 
 static Constant OODDAQ_PRECISION = 0.001
+static Constant OTHER_EPOCHS_PRECISION = 0.05
```

Comment:
Just out of interest:
Are the 0.05 a result from the generation sample clock of the stimset or a convenient choice specifically for this test?

Reply:
A mixture of playing around and deviation of stimset clock and DAQ clock.